stan/math/rev/mat.hpp

```diff
@@ -5,6 +5,7 @@
 #include <stan/math/rev/scal/meta/is_var.hpp>
 #include <stan/math/rev/scal/meta/partials_type.hpp>
 
+#include <stan/math/rev/mat/vectorize/apply_scalar_unary.hpp>
```

Comment:
There is. I tried changing the order and the tests failed. I agree that it's weird because of how the mix mat.hpp includes the other mat.hpp files.